### PR TITLE
Update Metricbeat data.json files

### PR DESCRIPTION
**Pull request by andrewkroh**
I ran `make generate-json` to update the autogenerated data.json samples on Linux.

### CODE REVIEW

**Comment by ruflin** [comment]
Thanks for doing this. We should have done this for quite some time :-( There are quite a few which are very outdated / wrong :-( Will go through it in detail tomorrow to see if this also shows some potential problems.

File: metricbeat/module/apache/status/_meta/data.json [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
+                "system": 16.39,
+                "user": 0.16
             },
             "hostname": "apache",
             "load": {
-                "1": 0.55,
-                "15": 0.31,
-                "5": 0.31
+                "1": 170.11,
+                "15": 77.62,
```
Comment (ruflin) [inline metricbeat/module/apache/status/_meta/data.json]: @andrewkroh I'm a bit surprised how high these values are

File: metricbeat/module/ceph/cluster_status/_meta/data.json [hunk 2]
```diff
@@ -1,55 +1,49 @@
...
+                "epoch": 18,
+                "full": false,
+                "in_osd_count": 1,
+                "nearfull": false,
+                "osd_count": 1,
+                "remapped_pg_count": 0,
+                "up_osd_count": 1
+            },
+            "pg": {
+                "avail_bytes": 50847313920,
```
Comment (ruflin) [inline metricbeat/module/ceph/cluster_status/_meta/data.json]: Seems like we need some `bytes` cleanup.

File: metricbeat/module/ceph/osd_df/_meta/data.json [hunk 3]
```diff
@@ -1,31 +1,25 @@
...
+        "name": "host.example.com"
+    },
+    "ceph": {
+        "osd_df": {
+            "available.byte": 49655580,
+            "device_class": "",
+            "id": 0,
+            "name": "osd.0",
+            "pg_num": 120,
+            "total.byte": 65792556,
```
Comment (ruflin) [inline metricbeat/module/ceph/osd_df/_meta/data.json]: `bytes` cleanup needed.

File: metricbeat/module/ceph/pool_disk/_meta/data.json [hunk 4]
```diff
@@ -1,29 +1,29 @@
...
-                },
-                "total": {
-                    "bytes": 6431965184
+                    "bytes": 47478734249
                 },
+                "objects": 0,
                 "used": {
-                    "bytes": 1391464448
+                    "bytes": 0,
+                    "kb": 0
```
Comment (ruflin) [inline metricbeat/module/ceph/pool_disk/_meta/data.json]: We should remove the second entry.

File: metricbeat/module/etcd/self/_meta/data.json [hunk 5]
```diff
@@ -1,43 +1,40 @@
...
-		    "pkgrate": 11.111234716807138
-		    }
-	    },
-	    "starttime": "2017-08-07T12:24:13.805809934+03:00",
-	    "state": "StateLeader"
-	}	    
+            "leaderinfo": {
+                "leader": "8e9e05c52164694d",
+                "starttime": "2017-12-07T07:20:33.241031712Z",
+                "uptime": "9m4.919923089s"
```
Comment (ruflin) [inline metricbeat/module/etcd/self/_meta/data.json]: That is a wiered format.

**Comment by andrewkroh** [comment]
+1 to fixing all those issues you identified.

**Comment by ruflin** [comment]
I created https://github.com/elastic/beats/issues/5858 for a follow up.

**Pull request merged**